runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -240,6 +264,23 @@ else if (state == GameState.LOGIN_SCREEN)
 		}
 	}
 
+	private void updateAttackStyle(int attackStyleIndex, int castingMode)
+	{
+		AttackStyle[] attackStyles = equippedWeaponType.getAttackStyles();
+		if (attackStyleIndex < attackStyles.length)
+		{
+			attackStyle = attackStyles[attackStyleIndex];
+			if (attackStyle == null)
+			{
+				attackStyle = OTHER;
```

Comment:
Is this a case which can ever happen?

Reply:
I mean there's plenty of `null` values in the [`WeaponType` constants](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/WeaponType.java?w=1#L43-L72). Maybe there's more I'm missing here though?

Reply:
@Adam- Seems like `null` values are not possible now? There's a few places where `attackStyle == null` checks exist that predate this PR